docs/pages/guides/testing-rsc.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Testing React Server Components
+sidebar_title: Testing RSC
+description: Learn about writing unit tests for React and React Native Server Components in Expo.
+platforms: ['android', 'ios', 'web']
+---
+
+> This guide refers to the experimental feature React Server Components which is still in development.
+
+React Server Components (RSC) are a new feature in React that allows you to build components that render on the server and can be hydrated on the client.
+
+Expo has support for testing React Server Components in your project.
+
+## Jest Testing
+
+Unlike client-based Jest testing, React Server Components run in a Node.js environment, meaning you can more closely emulate the server-side rendering environment with Jest on its own.
+
+### Setup
+
+While standard server rendering is web-only, Expo's universal React Server Components bundle custom server renderers for each platform. This means platform-extensions like `*.ios.js` and `*.native.ts` will be resolved when writing server components for an iOS app.
+
+`jest-expo` provides a couple different presets for testing server components:
+
+- `jest-expo/rsc/web`: An Android-only runner for React Server Components. Uses `*.web.js` and `*.js` files.
+- `jest-expo/rsc/ios`: An iOS-only runner for React Server Components. Uses `*.ios.js`, `*.native.js`, and `*.js` files.
+- `jest-expo/rsc/android`: An Android-only runner for React Server Components. Uses `*.android.js`, `*.native.js`, and `*.js` files.
+- `jest-expo/rsc`: A multi-runner which combines the above runners.
```

Comment:
```suggestion
| Runner                  | Description                                                                                                        |
| ----------------------- | ------------------------------------------------------------------------------------------------------------------ |
| `jest-expo/rsc/android` | An Android-only runner for RSC. Uses **\*.android.js**, **\*.native.js**, and **\*.js** files. |
| `jest-expo/rsc/ios`     | An iOS-only runner for RSC. Uses **\*.ios.js**, **\*.native.js**, and **\*.js** files.         |
| `jest-expo/rsc/web`     | A web-only runner for RSC. Uses **\*.web.js** and **\*.js** files.                            |
| `jest-expo/rsc`         | A multi-runner that combines the above runners.                                                                   |
```

Let's convert the above list items into a table to improve readability with the following changes
- There's a typo in the first item. It's for web.
- Let's use bold filenames as per our writing style guide.
- Use correct platform-order (Android, iOS, web) as per our writing style guide
- We can use the short form for React Server Components since that is already defined in the introduction for brevity.